### DISCUSSION TITLE
Add: Christian Television Network Channels

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#1190] [Show and tell]
### Channel Name

Christian Television Network / CTN

### Notes

Here Are Bunch Of CTN TV Channels

CTN
https://ctntv.getstreamhosting.com:1936/CTNTV/smil:CTNTV.smil/playlist.m3u8

CTN 22 Clearwater FL [West Center Florida] (WCLF-TV)
https://ctntv.getstreamhosting.com:1936/WCLF/WCLF/playlist.m3u8

CTN 48 Knoxville TN [East Tennessee] (WVLR-TV)
https://ctntv.getstreamhosting.com:1936/WVLR/smil:WVLR.smil/playlist.m3u8

CTN 16 Columbus GA [East Alabama - West Georgia] (WYBU-TV)
https://ctntv.getstreamhosting.com:1936/WYBU/WYBU/playlist.m3u8

CTN 61 Palm Beach FL [South Florida] (WFGC-TV)
https://rrsatrtmp.tulix.tv/livecdn631/myStream.sdp/playlist.m3u8

CTN 39 Nashville TN [Middle Tennessee] (WHTN-TV)
https://5dcabf026b188.streamlock.net/whtn39hd1/livestream/playlist.m3u8

CTN 18 Charleston SC [South Carolina] (WLCN-TV)
https://ctntv.getstreamhosting.com:1936/WLCN/WLCN/playlist.m3u8

CTN 45 Tallahassee FL (WVUP-TV)
https://ctntv.getstreamhosting.com:1936/WVUP/smil:WVUP.smil/playlist.m3u8

CTN 27.1 Chattanooga TN (WTNB-TV)
https://ctntv.getstreamhosting.com:1936/WTNB/WTNB/playlist.m3u8

CTN 16 Quincy IL (WTJR-TV)
https://ctntv.getstreamhosting.com:1936/WTJR/WTJR/playlist.m3u8

CTN 33 Pensacola FL [Gulf Coast] (WHBR-TV)
https://ctntv.getstreamhosting.com:1936/WHBR/WHBR/playlist.m3u8

CTN 45 Macon GA [Middle Georgia] (WGNM-TV)
https://ctntv.getstreamhosting.com:1936/WGNM/smil:WGNM.smil/playlist.m3u8

CTN 49 Tice FL [Southwest Florida] (WRXY-TV)
https://ctntv.getstreamhosting.com:1936/WRXY/WRXY/playlist.m3u8

CTN 17 Los Vegas NV (KEEN-TV)
https://ctntv.getstreamhosting.com:1936/KEEN/KEEN/playlist.m3u8

CTN 12.1 Houston TX (KHLM-TV)
https://ctntv.getstreamhosting.com:1936/KHLM/KHLM/playlist.m3u8

CTN 47 Tulsa OK (KWHB-TV)
https://ctntv.getstreamhosting.com:1936/KWHB/KWHB/playlist.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by what-a** [comment]
@R4IN2007 You always come up with good suggestions and stream links. No pressure, but you should become part of the team and start submitting pull requests.
